tensorflow/core/kernels/sparse_reduce_max_op_test.cc

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I'd prefer if you could leave out this file.  The reason we originally had a C++ test file for `SparseReduceSum` was to ease debugging.  Since now most of the kernels share the same logic, this is no longer needed.  The functional correctness of the new op can be covered by the more lightweight Python unit tests.